applications/openshift/api-server/api_server_tls_security_profile/tests/tls_security_profile.fail.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir -p /tmp/apis/config.openshift.io/v1/apiservers
+
+cat << EOF > /tmp/apis/config.openshift.io/v1/apiservers/cluster
+spec:
+    tlsSecurityProfile:
+        old: {}
+        type: Old
+EOF
```

Comment:
```suggestion
#!/bin/bash

# remediation = none

mkdir -p /kubernetes-api-resources/apis/config.openshift.io/v1/apiservers/

cat << EOF > /kubernetes-api-resources/apis/config.openshift.io/v1/apiservers/cluster
spec:
    tlsSecurityProfile:
        old: {}
        type: Old
EOF
```

Note the `# remediation = none` and the change in the `filepath`

Reply:
the `filepath` is probably not relevant here... we have an issue with the automatus github action check that we will be resolving on a different pull request: https://github.com/ComplianceAsCode/content/actions/runs/3460194333/jobs/5776501377 See `Run tests in a container...` step.

Reply:
comment incorporated, both filepath and remediation